src/Components/Nav/Nav.js

```diff
@@ -4,7 +4,7 @@ import './Nav.scss';
 class Nav extends React.Component {
   render() {
     return (
-      <nav className="Nav">
+      <div className="Nav">
```

Comment:
- 컴포넌트를 네이밍 할때만 첫글자를 대문자로 해주시고, 나머지는 소문자로 시작해주세요!
```suggestion
      <div className="nav">
```